cli/server.go

```diff
@@ -257,6 +257,10 @@ flags, and YAML configuration. The precedence is as follows:
 					continue
 				}
 
+				if opt.Value.String() == opt.Default {
+					continue
+				}
```

Comment:
Could you add a field to bigcli types similar to what cobra had for `.Changed()`?